test/unit/webgl/p5.RendererGL.js

```diff
@@ -36,7 +37,7 @@ suite('p5.RendererGL', function() {
       );
       done();
     });
-
+    */
```

Comment:
mostly because the whole concept of `curStrokeShader` has gone away. the shader that is used is determined at render time instead of being stored. the equivalent now would be `assert(myp5._renderer._getImmediateFillShader() === myp5._renderer._getImmediateFillShader());``

Reply:
that makes sense! I say just delete instead of commenting out.